features/src/gcc/install.sh

```diff
@@ -23,6 +23,11 @@ check_packages                  \
 GCC_VERSION_DEFAULT="$(gcc -dumpversion)";
 GCC_VERSION="${VERSION:-${GCC_VERSION_DEFAULT}}";
 
+if [[ "${GCC_VERSION}" == "15" ]]; then
+    apt-add-repository -y "deb https://ppa.launchpadcontent.net/ubuntu-toolchain-r/ppa/ubuntu plucky main";
+    apt-add-repository -y "deb https://ppa.launchpadcontent.net/ubuntu-toolchain-r/test/ubuntu plucky main";
+fi
```

Comment:
Is this worth future-proofing?

```suggestion
if [[ "${GCC_VERSION}" == "15" ]]; then
    VERSION_CODENAME="$(. /etc/os-release; echo ${VERSION_CODENAME})"
    apt-add-repository -y "deb https://ppa.launchpadcontent.net/ubuntu-toolchain-r/ppa/ubuntu $VERSION_CODENAME main";
    apt-add-repository -y "deb https://ppa.launchpadcontent.net/ubuntu-toolchain-r/test/ubuntu $VERSION_CODENAME main";
fi
```

Or maybe doing something more advanced, like checking if the desired `$GCC_VERSION` is installable via `apt-cache policy` and conditionally adding the repo?

Reply:
So, last time we just built using `23.04` and this time I was imagining it would be simpler to build by fetching from `plucky` the codename of the next release. When 26.04 comes around this would need to be removed. Either way it's kind of a hack.

Honestly I don't have good judgement for which way is better. Build with or pull from 25.04 to get gcc-15.

Reply:
What if we build the `gcc-15` images with ubuntu24.04? Would we need the changes here? Is there any reason we shouldn't?